owners/scripts/warm_cache.js

```diff
@@ -42,7 +42,7 @@ const {
   GITHUB_ACCESS_TOKEN,
   GITHUB_OWNER,
   GITHUB_REPOSITORY,
-} = process.env.CLOUD_STORAGE_BUCKET;
```

Comment:
😬

Reply:
Lol yep, not sure how that go there :eyes: